### PR TITLE
Cleanup Eslint JsDoc errors as well as warning appearing in freestyle fiori

### PR DESCRIPTION
**Pull request by ciaradaly2021**
# Issue

#270 

## Description

Updating the `model.js` and `component.js` files to ignore a jsdoc eslint error using relevant comments that was found whilst prettier updates were being made in Easy UI5. Issue originally only pointed out `model.js` but same issue was found in `Component.js`. Updating all files to fix eslint errors that appear for a lot of the js docs

### CODE REVIEW

File: packages/fiori-freestyle-writer/templates/basic/add/webapp/Component.js [hunk 1]
```diff
@@ -1,8 +1,15 @@
+/**
+ * eslint-disable @sap/ui5-jsdocs/no-jsdoc
+ */
```
Comment (devinea) [inline packages/fiori-freestyle-writer/templates/basic/add/webapp/Component.js:3]: @lboehm Is the `eslint-disable` needed or just the jsdoc below?

Comment (lboehm) [reply packages/fiori-freestyle-writer/templates/basic/add/webapp/Component.js:3]: @devinea Sorry for the late response, I had a busy week. I see you fixed it. Thank you very much!

**Review by tobiasqueck — COMMENTED** [review]
Why would this PR have changes to the `pnpm-lock.yaml`?

**Review by devinea — APPROVED** [review]
Thanks @IainSAP changes look good.

**Review by tobiasqueck — APPROVED** [review]

**Review by devinea — APPROVED** [review]

**Pull request merged**